_pandoc_filter/add_title.lua

```diff
@@ -0,0 +1,317 @@
+function fix_path (path)
+  return '.' .. path
+end
+
+
+function skip_include(el)
```

Comment:
Extremely confusing name, either rename to something that makes sense or add a comment